_posts/2016-03-09-release-2-6-0.markdown

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Release 2.7.0"
```

Comment:
Good catch !
It's what happen by writing a blog post just after create the new milestone ;)